docs/manual/src/internals/foreign_calls.md

```diff
@@ -0,0 +1,88 @@
+# Rust -> Foreign calls
+
+How can Rust code make a call into the foreign language?
+The [lifting and lowering](./lifting_and_lowering.md) docs describe this at a high-level, this document will explore the details.
+
+This document only describes non-async calls.
+See the [Async FFI](./async-ffi.md) docs for a description of async calls.
+
+## VTables
+
+Calling into foreign code has 2 major differences from [calling Rust code](./rust_calls.md):
+
+* Foreign code is only called via an interface.
+* The Rust code does not have access to a library with functions it can call.
```

Comment:
I would like to reword this _slightly_. Like what does it mean? Does it mean that "Rust doesn't know the functions it can call on the foreign language side". 

It clashes a bit with the next paragraph, which says that the foreign code registers functions Rust can call?

I think I know what you mean, but I had to read it a few times to get it.